README.md

```diff
@@ -1,6 +1,31 @@
+<div align="center">
+
 # Kubecost Helm Chart
 
-This repository contains the official Helm chart for **Kubecost v3+**, providing cost monitoring and optimization for Kubernetes clusters.
+[![GitHub stars](https://img.shields.io/github/stars/kubecost?style=social)](https://github.com/kubecost/kubecost/stargazers)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Chart Publish](https://github.com/kubecost/release-scripts/actions/workflows/release-kubecost-v3.yaml/badge.svg)]()
+[![Latest Release](https://img.shields.io/github/v/release/kubecost/kubecost)]()  
+
+<div align="left">
+
+## Kubecost
+
+Kubecost gives teams visibility into current and historical Kubernetes spend and resource allocation. These models provide cost transparency in Kubernetes environments that support multiple applications, teams, departments, etc.
+
+To see more on the functionality of the full Kubecost product, please visit the [features page](https://www.apptio.com/products/kubecost/?src=kc-com#features) on our website.
+
+Some of the features of Kubecost include:
+
+- Real-time cost allocation by Kubernetes Service, Deployment, Namespace, label, StatefulSet, DaemonSet, Pod, and container
+- Dynamic asset pricing enabled by integrations with AWS, Azure, and GCP billing APIs
+- Supports on-premises Kubernetes clusters with custom pricing sheets
+- Allocation for in-cluster resources like CPU, GPU, memory, and persistent volumes
+- Allocation for AWS and GCP out-of-cluster resources like RDS instances and S3 buckets with key (optional)
+
+## Getting Started
```

Comment:
This is quite outdated. "Kubernetes 1.8+". To keep things clear and concise, I'd recommend deleting this section. Thoughts?